doc/usage/installation.rst

```diff
@@ -192,12 +192,15 @@ When installing Sphinx using pip,
 it is highly recommended to use *virtual environments*,
 which isolate the installed packages from the system packages,
 thus removing the need to use administrator privileges.
-To create a virtual environment in the ``.venv`` directory,
-use the following command.
+To create a virtual environment in the ``~/.venv`` directory
+and install Sphinx, use the following commands.
 
 ::
 
-   $ python -m venv .venv
+   $ cd ~
+   $ python3 -m venv .venv
+   $ ~/.venv/bin/python3 -m pip install --upgrade pip
+   $ ~/.venv/bin/python3 -m pip install sphinx
```

Comment:
Windows doesn't use `python3`:

```suggestion
   $ python3 -m venv .venv
   $ .venv/bin/python -m pip install --upgrade pip
   $ .venv/bin/python -m pip install sphinx
```

I assume `python` can be used in venvs for python3 in posix systems.